### PR TITLE
[DatePicker] fix validationOnLoad prop is ignored

### LINKED ISSUE #25318: [Bug]: DatePicker performs validation on load, despite validateOnLoad = false.
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (16) x64 AMD Ryzen 7 PRO 4750U with Radeon Graphics
    Memory: 14.07 GB / 63.37 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (106.0.1370.37)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

_No response_

### Reproduction

https://codepen.io/vger/pen/NWMZwWb

## Actual Behavior
On initial render the DatePicker is shown and validation is performed, despite validateOnLoad = false for textField.

## Expected Behavior
I expect it not to perform the validation on load, but after either filling out the input field or on blur.

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by Svudec**
## Previous Behavior

On initial render the DatePicker is shown and validation is performed, despite `validateOnLoad = false` in `textField` props.

## New Behavior

If `validateOnLoad = false` in `textField` props passed to DatePicker, validation won't be performed on first render

## Related Issue(s)

* Fixes #25318

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9'>2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25961&commitId=b802266c9a2d676ab41a3c49e363392504b4152a&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount_base.html">1892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount.html">2078</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/BaseButton-mount_base.html">1121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/BaseButton-mount.html">1095</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Breadcrumb-mount_base.html">2795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Breadcrumb-mount.html">2984</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Checkbox-mount_base.html">2420</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Checkbox-mount.html">2566</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/CheckboxBase-mount_base.html">2141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/CheckboxBase-mount.html">2354</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ChoiceGroup-mount_base.html">4174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ChoiceGroup-mount.html">4402</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ComboBox-mount_base.html">1266</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ComboBox-mount.html">1186</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/CommandBar-mount_base.html">9815</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/CommandBar-mount.html">9697</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ContextualMenu-mount_base.html">11961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ContextualMenu-mount.html">12080</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DefaultButton-mount_base.html">1301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DefaultButton-mount.html">1242</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRow-mount_base.html">3479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRow-mount.html">3308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRowFast-mount_base.html">3432</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRowFast-mount.html">3468</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRowNoStyles-mount_base.html">3126</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DetailsRowNoStyles-mount.html">3414</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Dialog-mount_base.html">3073</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Dialog-mount.html">2907</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DocumentCardTitle-mount_base.html">522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/DocumentCardTitle-mount.html">527</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Dropdown-mount_base.html">2955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Dropdown-mount.html">2970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/FocusTrapZone-mount_base.html">1858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/FocusTrapZone-mount.html">1918</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/FocusZone-mount_base.html">1802</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/FocusZone-mount.html">1793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount_base.html">1892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount.html">2078</a></td>
            <td>2</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-virtual-rerender_base.html">1069</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-virtual-rerender.html">1108</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedList-virtual-rerender-with-unmount.html">1671</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-mount_base.html">511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-mount.html">548</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-virtual-rerender_base.html">491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-virtual-rerender.html">524</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">564</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">525</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/IconButton-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/IconButton-mount.html">1719</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Label-mount_base.html">716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Label-mount.html">662</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Layer-mount_base.html">4074</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Layer-mount.html">4099</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Link-mount_base.html">767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Link-mount.html">798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/MenuButton-mount_base.html">1611</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/MenuButton-mount.html">1546</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/MessageBar-mount_base.html">2119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/MessageBar-mount.html">2198</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Nav-mount_base.html">3088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Nav-mount.html">2951</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/OverflowSet-mount_base.html">1245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/OverflowSet-mount.html">1379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Panel-mount_base.html">2531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Panel-mount.html">2353</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Persona-mount_base.html">1218</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Persona-mount.html">1202</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Pivot-mount_base.html">1595</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Pivot-mount.html">1513</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/PrimaryButton-mount_base.html">1440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/PrimaryButton-mount.html">1506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Rating-mount_base.html">6480</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Rating-mount.html">6646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SearchBox-mount_base.html">1395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SearchBox-mount.html">1361</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Shimmer-mount_base.html">2594</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Shimmer-mount.html">2641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Slider-mount_base.html">1868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Slider-mount.html">1966</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SpinButton-mount_base.html">4418</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SpinButton-mount.html">4341</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Spinner-mount_base.html">735</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Spinner-mount.html">733</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SplitButton-mount_base.html">2824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SplitButton-mount.html">2814</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Stack-mount_base.html">788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Stack-mount.html">820</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/StackWithIntrinsicChildren-mount_base.html">2248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/StackWithIntrinsicChildren-mount.html">2263</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/StackWithTextChildren-mount_base.html">4511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/StackWithTextChildren-mount.html">4414</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SwatchColorPicker-mount_base.html">9707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/SwatchColorPicker-mount.html">9886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TagPicker-mount_base.html">2509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TagPicker-mount.html">2427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TeachingBubble-mount_base.html">86409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TeachingBubble-mount.html">80505</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Text-mount_base.html">770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Text-mount.html">730</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TextField-mount_base.html">1606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/TextField-mount.html">1573</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-mount_base.html">1487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-mount.html">1417</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-virtual-rerender_base.html">1049</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-virtual-rerender.html">997</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2000</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2031</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Toggle-mount_base.html">1049</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/Toggle-mount.html">1123</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/buttonNative-mount_base.html">508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25961/perf-test/buttonNative-mount.html">534</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by Svudec** [comment]
@khmakoto Do you think that behavior of other `TextField` validation props (`validateOnFocusIn` and `validateOnFocusOut`) should be implemented in the DatePicker?

**Review by conniec218 — APPROVED** [review]

**Comment by khmakoto** [comment]
@Svudec if you could send a separate PR for those 2 other props that would be amazing.

File: change/@fluentui-react-f7e63c25-beed-4e4a-bdea-50c96a909588.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "[DatePicker] onLoadValidation can be turned off through textField props ",
```
Comment (khmakoto) [inline change/@fluentui-react-f7e63c25-beed-4e4a-bdea-50c96a909588.json:3]:
```suggestion
  "comment": "fix(DatePicker): onLoadValidation can be turned off through textField props.",
```

File: change/@fluentui-react-f7e63c25-beed-4e4a-bdea-50c96a909588.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (khmakoto) [inline change/@fluentui-react-f7e63c25-beed-4e4a-bdea-50c96a909588.json:2]:
```suggestion
  "type": "patch",
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**